File: Employee.java
import java.util.ArrayList;
class Employee {
    private String nombre;
    private ArrayList<BankAccount> accounts;

    public Employee(String nombre) {
        this.nombre = nombre;
        this.accounts = new ArrayList<>();
    }

    public void agregarCuenta(BankAccount account) {
        accounts.add(account);
    }

    public String getNombre() {
        return nombre;
    }

    public ArrayList<BankAccount> getAccounts() {
        return accounts;
    }
}

File: Employee.py
class Employee:
    def __init__(self, nombre):
        self.nombre = nombre
        self.accounts = []

    def agregar_cuenta(self, account):
        self.accounts.append(account)

    def get_nombre(self):
        return self.nombre

    def get_accounts(self):
        return self.accounts